### PR TITLE
summary: updated summary with 19.0 changes

### PR DESCRIPTION
**Pull request by systay**
## Description
Updates the summary information with new changes added in v19

## Related Issue(s)
https://github.com/vitessio/vitess/issues/15131

## Checklist
-   [x] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [x] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on CI?
-   [x] Documentation was added or is not required

### CODE REVIEW

File: changelog/19.0/19.0.0/summary.md [hunk 1]
```diff
@@ -184,6 +184,29 @@ mysql> show warnings;
 
 A new flag `--vtcombo-bind-host` has been added to vttestserver that allows the users to configure the bind host that vtcombo uses. This is especially useful when running vttestserver as a docker image and you want to run vtctld commands and look at the vtcombo `/debug/status` dashboard.
 
+### <a id="vttestserver"/>New lock syntax
```
[the remark this answers is not in the document]
Comment (systay) [reply changelog/19.0/19.0.0/summary.md]: 👍 Thanks for reviewing!

File: go/vt/vtgate/planbuilder/collations_test.go [hunk 2]
```diff
@@ -20,12 +20,11 @@ import (
 	"fmt"
 	"testing"
 
-	"vitess.io/vitess/go/test/vschemawrapper"
```
Comment (GrahamCampbell) [inline go/vt/vtgate/planbuilder/collations_test.go:23]: What is this change doing here?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: go/vt/vtgate/planbuilder/collations_test.go (hunk 2 above)
Comment (systay) [reply go/vt/vtgate/planbuilder/collations_test.go:23]: *sigh*

My IDE will auto-cleanup imports when a file is open. I must have had this file open while I switched branches. I fix.

File: changelog/19.0/19.0.0/summary.md [hunk 3]
```diff
@@ -184,6 +184,29 @@ mysql> show warnings;
...
+### <a id="cte-support"/>Support for non-recursive CTEs
+
+Common table expressions that are not recursive can now be used. 
+
+```sql
+with userCount as (
+    select id, count(*) as nr from user group by id)
+select ref.col, userCount.nr
+from ref join userCount on ref.user_id = userCount.id
+```
```
Comment (frouioui) [inline changelog/19.0/19.0.0/summary.md:209]: We must add those links to the summary at the start of this file

**Review by frouioui — APPROVED** [review]

**Review by shlomi-noach — APPROVED** [review]

**Pull request merged**